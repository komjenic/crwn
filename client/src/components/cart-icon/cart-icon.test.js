import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import { CartIcon } from './cart-icon.component';

it('expect to render CartIcon component', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<CartIcon />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
});
