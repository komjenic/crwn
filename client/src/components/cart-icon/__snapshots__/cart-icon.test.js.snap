// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CartIcon component 1`] = `
<ForwardRef(styled.div)>
  <ForwardRef(Styled(Component)) />
  <ForwardRef(styled.span) />
</ForwardRef(styled.div)>
`;
